### ISSUE TITLE
AIMed data processing

### ISSUE THREAD

**Issue by Hrant-Khachatrian** [ylhsieh/ppi_lstm_rnn_keras#1]
Thanks for sharing the code of your model.

Can you please share the details (or code) for processing AIMed XMLs? In particular, how do you handle nested protein names when generating the candidates?

Also, as far as I understand from the paper, you do not use symmetric pairs. E.g. if A interacts with B, you do not add "B interacts to A" in the training set. Is there any reason for that?

Thanks in advance.

**Comment by Hrant-Khachatrian** [comment]
One more question: which version of AIMed you used as an input? 

There is one original [compressed file](ftp://ftp.cs.utexas.edu/pub/mooney/bio-data/). We tried to extract positive and negative samples from that and could get 4163, 4196, 4317 or 4351 negative samples (depending on the way we handle nested tags). None of them match 4834 as reported in your paper.

We have found another (XML) version of AIMed, which is [modified to become more consistent with other datasets](http://mars.cs.utu.fi/PPICorpora/eval-standard.html). There are many differences, but the most notable one is that sometimes two consecutive sentences are merged into a single "entry", and the number of negative samples become larger. In fact, one of our algorithms extracted exactly 4834 negative samples from that XML. The problem with this approach is that the XML already has a fixed train/test split, which is not what you did in this paper. 

So we still can't figure out how the data was created for this model..
